Marlin/UltiLCD2_menu_material.cpp

```diff
@@ -853,7 +853,7 @@ static void lcd_material_settings_details_callback(uint8_t nr)
     buffer[0] = '\0';
     if (nr == 0)
     {
-        return;
+        strcpy(buffer, material[active_extruder].name);
```

Comment:
Might be safer to use the strcpy variant which you can give a limit to how far it uses the source and dest buffers.

Reply:
Will replace the call by strlcpy which has the advantage over strncpy that it will always zero terminate the string.